src/schema/basic_slots.yaml

```diff
@@ -217,7 +217,11 @@ slots:
   biomaterial_purity:
     domain: ProcessedSample
     range: QuantityValue
-
+  
+  is_best_protein:
+    description: Slot to store a boolean value. If populated with a  value of true the gene function is the best protein hit, if false it is not the best protein hit.
```

Comment:
There are two spaces between "a" and "value" here.

Reply:
I think "Slot to store a boolean value" is redundant, given the `range: boolean` below.

Reply:
Yes, this needs to be rewritten. What does it mean for a function to be a best protein hit? When I think of "best protein hit" I think of matching raw data, like mass spectra or nucleotide sequences, to some database. The assignment of a function also has uncertainly of its own, based on inference with something like a HMM.

What class(es) are intended to be in the domain of `is_best_protein`?

If this slot is just about identifying the protein inference that best matches the observed data, the description should be something like "True when the named protein is the best explanation of teh observed data" or something like that.